configuration/transitions.md

```diff
@@ -244,21 +244,35 @@ Implements muting/unmuting of persons and places. Supports multiple forms for ea
+- updates target contact and all its descendants<sup>[1]</sup>, setting the `muted` property equal to the current `date` in ISO format<sup>[2]</sup>
+- adds a `muting_history` entry to Sentinel `info` docs for every updated contact<sup>[7]</sup>
+- updates all connected registrations<sup>[3]</sup>, changing the state of all unsent<sup>[4]</sup> `scheduled_tasks` to `muted`
 
 Unmuting action:
 
-- updates target contact's topmost muted ancestor<sup>[3]</sup> and all its descendants, setting the `muted` property to `false`
-- updates all connected registrations<sup>[1]</sup>, changing the state of all present/future<sup>[4]</sup> `muted` `scheduled_tasks` to `scheduled`
+- updates target contact's topmost muted ancestor<sup>[1][5]</sup> and all its descendants, removing the `muted` property
+- adds a `muting_history` entry to Sentinel `info` docs for every updated contact<sup>[7]</sup>
+- updates all connected registrations<sup>[3]</sup>, changing the state of all present/future<sup>[6]</sup> `muted` `scheduled_tasks` to `scheduled`
 
-[1] target contact and descendants' registrations  
-[2] `scheduled_tasks` having either `scheduled` or `pending` state  
-[3] because the muted state is inherited, unmuting cascades upwards to the highest level muted ancestor. If none of the ancestors is muted, unmuting cascades downwards only.  
-[4] scheduled tasks which are due today or in the future. All `scheduled_tasks` with a due date in the past will remain unchanged.   
+[1] Contacts that are already in the correct state are skipped. This applies to updates to the contact itself, updates to the Sentinel `muting_history` and to the connected registrations (registrations of a contact that is already in the correct state will not be updated).    
+[2] The date represents the moment Sentinel has processed the muting action   
+[3] target contact and descendants' registrations  
+[4] `scheduled_tasks` being in either `scheduled` or `pending` state  
+[5] Because the muted state is inherited, unmuting cascades upwards to the highest level muted ancestor. If none of the ancestors is muted, unmuting cascades downwards only.  
+[6] `scheduled_tasks` which are due today or in the future. All `scheduled_tasks` with a due date in the past will remain unchanged.    
+
+##### [7] Muting history
+Each time the `muted` state of a contact changes, an entry is added to a `muting_history` list saved in Sentinel `info` docs (stored as an array property with the same name).     
+Entries in `muting_history` contain the following information:
+
+| Property | Description |
+| --- | --- | 
+| muted | Boolean representing the muted state |
+| date | Date in ISO Format |
+| report_id | An `_id` reference to the report that triggered the action |
 
 #### Configuration
-Configuration is stored in the `muting` field of the settings.
+Configuration is stored in the `muting` field of the `settings` doc.
```

Comment:
We should make sure it is clear that for configurers this is done in `app_settings.json`.

Reply:
Updated to mention app_settings specifically.